integration/cli-app/.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+.zos.session
+zos.*
+workdir/build/*
+workdir/node_modules
+workdir/zos*
+workdir/package*
+workdir/contracts/*
```

Comment:
What are these directories?

Reply:
`workdir` is the working directory where the test project is executed, so it is mostly gitignored